CONTRIBUTING.adoc

```diff
@@ -66,9 +66,9 @@ export OS_OUTPUT_GOPATH=1
 
         $ mkdir -p $GOPATH/src/github.com/openshift
         $ cd $GOPATH/src/github.com/openshift
-        $ git clone git://github.com/<forkid>/origin  # Replace <forkid> with the your github id
+        $ git clone https://github.com/<forkid>/origin  # Replace <forkid> with the your github id
```

Comment:
Hi liggitt and stevekuznetsov ,
git://... doesn't work in my VM, but https://  can work fine. 
I am no sure that  it may be because of proxy in the company.  
So, I suggest that it is a general method to use htttps for git clone which is shown to users in documents. 

$ env | grep http                            
http_proxy=http://proxysh.zte.com.cn:80/
https_proxy=https://proxysh.zte.com.cn:80/
$ git clone git://github.com/openshift/origin  
Cloning into 'origin'...
fatal: unable to connect to github.com:
github.com[0: 192.30.253.112]: errno=Connection timed out
$ rm -rf origin/
$ git clone https://github.com/openshift/origin
Cloning into 'origin'...
remote: Counting objects: 176423, done.
remote: Compressing objects: 100% (9/9), done.
remote: Total 176423 (delta 2), reused 0 (delta 0), pack-reused 176414
Receiving objects: 100% (176423/176423), 298.21 MiB | 1.70 MiB/s, done.
Resolving deltas: 100% (103827/103827), done.
Checking out files: 100% (10736/10736), done.

@liggitt @stevekuznetsov PTAL
Thanks